dag_in_context/src/schedule.egg

```diff
@@ -0,0 +1,37 @@
+(run-schedule
```

Comment:
Why do we need this file and why is it a new file?

Reply:
forget to delete this